README.md

```diff
@@ -322,3 +322,43 @@ You can decode custom types the same way, as long as the type also conforms to
 `JSONDecodable`.
 
 For more examples on how to use Argo, please check out the tests.
+
+Contributing
+------------
+
+See the [CONTRIBUTING] document. Thank you, [contributors]!
+
+[CONTRIBUTING]: CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/Argo/graphs/contributors
+
+Need Help?
+----------
+
+We offer 1-on-1 coaching. We can help you with functional programming in Swift,
+get started writing unit tests, and convert from Objective-C to Swift.
+[Get in touch].
+
+[Get in touch]: http://coaching.thoughtbot.com/ios/?utm_source=github
+
+License
+-------
+
+Argo is Copyright (c) 2015 thoughtbot, inc. It is free software, and may be
+redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
+
+About
+-----
+
+![thoughtbot](https://thoughtbot.com/logo.png)
```

Comment:
I might even move this above coaching. Coaching is cool, but will never be our primary business. Thoughts?